finch.yaml

```diff
@@ -151,9 +151,14 @@ provision:
     sudo chown $USER /mnt/lima-finch
     mkdir -p /mnt/lima-finch/containerd
     mkdir -p /mnt/lima-finch/nerdctl
+    mkdir -p /mnt/lima-finch/cni-config
+    mkdir -p /mnt/lima-finch/cni-local
     mkdir -p ~/.local/share/nerdctl
+    mkdir -p ~/.config/cni
     sudo mount --bind /mnt/lima-finch/containerd ~/.local/share/containerd
     sudo mount --bind /mnt/lima-finch/nerdctl ~/.local/share/nerdctl
+    sudo mount --bind /mnt/lima-finch/cni-config ~/.config/cni
+    sudo mount --bind /mnt/lima-finch/cni-local ~/.local/share/cni
```

Comment:
Maybe we can add some reference links for each line. Take `~/.local/share/nerdctl` for example, the reference link would be https://github.com/containerd/nerdctl/blob/main/docs/dir.md#data.

Reply:
I like this in theory, but I'm not too sure there is a good reference for the others. "~/.local/share/cni" turns up no results on Google, for example. I can add them on a best-effort basis.

Reply:
That's true. If we don't add `~/.local/share/cni`, what errors will be there? Maybe we start from there to see which part of code is using it. The link doesn't have to be a user-facing doc; it could be a permalink to the relevant code if that's what we can get.

Reply:
I was able to find [this snippet of code in `containerd-rootless.sh`](https://github.com/containerd/nerdctl/blob/main/extras/rootless/containerd-rootless.sh#L145-L150) that references the cni and containerd user data directories, so that should work.